### PR TITLE
Development

### PR DESCRIPTION
**Pull request by gervanpn**
Please review our project and approve

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                prespicatis,repudiandae nulla animi iste vel, praesentium repellendus
+                molestais aliquid consequatur, earum rem qui error voluptates eius praesentium
+                cosequuntur!
+            </p>
+        </div>
+    </div>
+
+</body>
+
+</html>
```
Comment (JuliaDunayeva) [inline About.html]: you do not need extra lines there

File: Contact.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  </head>
+  <body>
+    <div class="togglearea">
+        <div class="logo">
+            <h3 class="font-one-md">Clean Blog</h3>
+        </div>
+        <label for="toggle" class="font-one-sm">Menu
+            <hr>
+            <hr>
+            <hr>
```
Comment (JuliaDunayeva) [inline Contact.html:18]: You could use an icon here

File: Contact.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+      <meta name="viewport" content="width=divice-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800&display=swap" rel="stylesheet">
```
Comment (JuliaDunayeva) [inline Contact.html:7]: Nice!

File: Posts.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=divice-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800&display=swap" rel="stylesheet">
+    <title>Clean Blog - Posts</title>
+</head>
+
+<body>
```
Comment (JuliaDunayeva) [inline Posts.html]: You do not need empty lines here

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,257 @@
+body,
+html {
+    padding: 0px;
+    margin: 0px;
+}
+
+h1,
+h2 {
```
Comment (JuliaDunayeva) [inline styles.css]: have selectors in one line ex h1, h2

**Review by JuliaDunayeva — CHANGES_REQUESTED** [review]

**Review by gervanpn — COMMENTED** [review]
Made changes as per requests

**Comment by gervanpn** [comment]
Merging final review and changes to master

**Pull request merged**